### PR TITLE
logging: update documentation

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
Update the logging integration docs for the new required attributes `dd.service` and `dd.env`, `dd.version`

### CODE REVIEW

File: ddtrace/contrib/logging/__init__.py [hunk 1]
```diff
@@ -40,7 +45,9 @@ def hello():
     from ddtrace import tracer
 
     FORMAT = ('%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] '
-              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] '
+              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s'
```
Comment (brettlangdon) [inline ddtrace/contrib/logging/__init__.py]: must be `[{key:value} dd.trace_id={id} dd.span_id={id}]`, `dd.trace_id=0 dd.span_id=0` must always be last,

Comment (Kyle-Verhoog) [reply ddtrace/contrib/logging/__init__.py]: oh.. we should document this too then..

File: ddtrace/contrib/logging/__init__.py [hunk 2]
```diff
@@ -33,14 +38,18 @@ def hello():
...
+Note that ``dd.trace_id`` and ``dd.span_id`` must come last::
 
     from ddtrace import patch_all; patch_all(logging=True)
     import logging
     from ddtrace import tracer
 
     FORMAT = ('%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] '
-              '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] '
+              '[dd.service=%(dd.service)s dd.env=%(dd.env)s '
+              'dd.version=%(dd.version)s] '
```
Comment (brettlangdon) [inline ddtrace/contrib/logging/__init__.py]: you have an extra `]` here `dd.version=%(dd.version)s]`

Comment (Kyle-Verhoog) [reply ddtrace/contrib/logging/__init__.py]: ugh dang it..

Comment (Kyle-Verhoog) [reply ddtrace/contrib/logging/__init__.py]: good catch!

**Pull request merged**